cmd/src/users_prune.go

```diff
@@ -190,25 +246,27 @@ type UserToDelete struct {
 }
 
 // Verify user wants to remove users with table of users and a command prompt for [y/N]
-func confirmUserRemoval(usersToRemove []UserToDelete) (bool, error) {
-	fmt.Printf("Users to remove from instance at %s\n", cfg.Endpoint)
-	t := table.NewWriter()
-	t.SetOutputMirror(os.Stdout)
-	t.AppendHeader(table.Row{"Username", "Email", "Days Since Last Active"})
-	for _, user := range usersToRemove {
-		if user.User.Email != "" {
-			t.AppendRow([]interface{}{user.User.Username, user.User.Email, user.DaysSinceLastUse})
-			t.AppendSeparator()
-		} else {
-			t.AppendRow([]interface{}{user.User.Username, "", user.DaysSinceLastUse})
-			t.AppendSeparator()
+func confirmUserRemoval(usersToDelete []UserToDelete, totalUsers int, daysThreshold int, displayUsers bool) (bool, error) {
+	if displayUsers {
+		fmt.Printf("Users to remove from %s\n", cfg.Endpoint)
+		t := table.NewWriter()
+		t.SetOutputMirror(os.Stdout)
+		t.AppendHeader(table.Row{"Username", "Email", "Days Since Last Active"})
+		for _, user := range usersToDelete {
+			if user.User.Email != "" {
+				t.AppendRow([]interface{}{user.User.Username, user.User.Email, user.DaysSinceLastUse})
+				t.AppendSeparator()
+			} else {
+				t.AppendRow([]interface{}{user.User.Username, "", user.DaysSinceLastUse})
+				t.AppendSeparator()
+			}
 		}
+		t.SetStyle(table.StyleRounded)
+		t.Render()
 	}
-	t.SetStyle(table.StyleRounded)
-	t.Render()
 	input := ""
 	for strings.ToLower(input) != "y" && strings.ToLower(input) != "n" {
-		fmt.Printf("Do you  wish to proceed with user removal [y/N]: ")
+		fmt.Printf("%v of %v users were inactive for more than %v days on %v.\nDo you  wish to proceed with user removal [y/N]: ", len(usersToDelete), totalUsers, daysThreshold, cfg.Endpoint)
```

Comment:
I think `totalUsers` here is misleading, since it includes soft-deleted users. I wouldn't include that.

Reply:
Very good point, I don't think its worth trying to count users returned with null `deletedAt` just for this either, or make a graphql request to pull total users on the instance